client/src/components/PacmanGame/PacmanBoard.jsx

```diff
@@ -34,7 +34,8 @@ const PacmanBoard = ({ gridState, gridSize, pacman }) => (
 );
 
 PacmanBoard.propTypes = {
-  gridState: PropTypes.number.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Don't disable. Use `PropTypes.instanceOf(Array).isRequired;` instead,

Reply:
I'll specify the datatype instead. Using `PropTypes.instanceOf(Array)` would be as vague as ` PropTypes.array`